content-new/docs/api/getting-started.mdx

```diff
@@ -343,26 +344,40 @@ Webhook responses contain a 200 HTTP status code which includes a `Nonce` number
 | `Nonce`            | int         | The value that you receive in the webhook                                   |
 | `200 `             | int         | HTTP response status code                                                   |
 
-### Responding to a webhook
+## Handling a webhook
 
-The webhook sent by ClearBank acts as a verification mechanism.
-You verify that it is ClearBank that has sent you the webhook and subsequently, ClearBank needs to verify that you have successfully received the webhook (based on your response acknowledging receipt).
+The webhook sent by ClearBank® acts as a verification mechanism.
+It is important that you verify that it is ClearBank® that has sent you the webhook and subsequently, ClearBank® verifies that you have successfully received the webhook (based on your response acknowledging receipt).
 
-This verification occurs when you return a `200` HTTP response, which includes the `Nonce` number in the `response body`  and your `DigitalSignature`.
-If your response is invalid, then the webhook is considered to have failed. ClearBank will send the webhook every 15 minutes for 24 hours unless a successful, valid response is returned by you.
+We have provided an example C# controller in the [ClearBank fi-api-101 Github Repository](https://github.com/clearbank/fi-api-101) which shows how to calculate the Hash value and verify the digital signature.
 
-**To verify the integrity of the webhook received from ClearBank**
-- Download a Public Key specific to your institution by clicking the **Download Public Key** button on the Webhook Management page of the Institution portal.
+The handling of a webhook can be thought of in three different parts: Verification, Response and finally Processing.
+
+### Verification
+
+Step 1 is webhook verification. Here you must verify that ClearBank® is the true source of the webhook, and that it hasn't been tampered with.
+
+To verify the integrity of the webhook received from ClearBank®:
+- Download a Public Key that is specific to your institution by clicking the **Download Public Key** button on the Webhook Management page of the Institution portal.
```

Comment:
There is no need to mention that it is specific to the institution. It is just an internal implementation details
```suggestion
- Download a Public Key by clicking the **Download Public Key** button on the Webhook Management page of the Institution portal.
```